features/graphql/filters.feature

```diff
@@ -107,11 +107,33 @@ Feature: Collections filtering
       }
     }
     """
-    And the JSON node "data.dummies.edges[0].node.relatedDummies.edges" should have 0 elements
-    And the JSON node "data.dummies.edges[1].node.relatedDummies.edges" should have 0 elements
+    Then the JSON node "data.dummies.edges[0].node.relatedDummies.edges" should have 0 element
```

Comment:
Actually, zero is always followed by the plural form in English.

Reply:
Ho yes you're right. But it's in master now, there will be some conflicts. Do you want to keep them anyway?

Reply:
We can resolve the conflicts when merging back to `master`.